plugins/fe_events_moodle/classes/event/fe_event.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
```

Comment:
This line may be triggering an error in the CodeChecker:

![image](https://user-images.githubusercontent.com/839025/180849738-96e47984-f503-4a77-b218-8ab4b8ad26cb.png)